mev-relay-rs/src/relay.rs

```diff
@@ -160,7 +190,8 @@ impl BlindedBlockProvider for Relay {
     }
 
     async fn fetch_best_bid(&self, bid_request: &BidRequest) -> Result<SignedBuilderBid, Error> {
-        validate_bid_request(bid_request)?;
+        let is_registered_public_key = self.validator_registry.get_validator_index(&bid_request.public_key);
```

Comment:
if you just want to pass in a boolean here, I'd suggest this:
```suggestion
        let is_registered_public_key = self.validator_registry.get_validator_index(&bid_request.public_key).is_some();
```